install/lib/class.installer.php

```diff
@@ -318,7 +318,7 @@ private function checkConfiguration()
             // Check the database credentials
             elseif ($db && $db->isConnected()) {
                 // Incorrect MySQL version
-                if (version_compare($db->getVersion(), '5.5', '<')) {
+                if (\Composer\Semver\Comparator::lessThan($db->getVersion(), '5.6')) {
                     $errors['database-incorrect-version']  = array(
                         'msg' => 'MySQL Version is not correct. '. $version . ' detected.',
                         'details' => __('Symphony requires %1$s or greater to work, however version %2$s was detected. This requirement must be met before installation can proceed.', array('<code>MySQL 5.5</code>', '<code>' . $version . '</code>'))
```

Comment:
The text will say that you need at least MySQL 5.5

Reply:
Thanks for keeping an eye on this!

Reply:
@michael-e np!